tools/speech_data_explorer/data_explorer.py

```diff
@@ -83,6 +83,7 @@ def parse_args():
     )
     parser.add_argument('--vocab', help='optional vocabulary to highlight OOV words')
     parser.add_argument('--port', default='8050', help='serving port for establishing connection')
+    parser.add_argument('--base_url', default='/', help='base url pathname')
```

Comment:
this change needed if someone wants to access SDE remotely, right?
I mean if these are generic then we can submit a PR to Nvidia/NeMo project also.

Reply:
This change is needed only when the pages are served behind a proxy path. For eg: Instead of running the server at 127.0.01:8050 (default root is '/'), if we tend to serve it from 127.0.0.1:8050/http/path/sde (now the root will be '/http/path/sde'), we need to pass the proxified path which is ''/http/path/sde' to '--base_url'